File: src/main/java/com/github/njuro/jard/ban/BanMapper.java
package com.github.njuro.jard.ban;

import com.github.njuro.jard.ban.dto.BanDto;
import com.github.njuro.jard.base.BaseMapper;
import com.github.njuro.jard.user.UserMapper;
import org.mapstruct.Mapper;

@Mapper(config = BaseMapper.Config.class, uses = UserMapper.class)
public interface BanMapper extends BaseMapper<Ban, BanDto> {}

File: src/main/kotlin/com/github/njuro/jard/rewrite/ban/BanMapper.kt
package com.github.njuro.jard.rewrite.ban

import com.github.njuro.jard.rewrite.ban.dto.BanDto
import com.github.njuro.jard.rewrite.base.BaseMapper
import com.github.njuro.jard.rewrite.user.UserMapper
import org.mapstruct.Mapper

@Mapper(config = BaseMapper.Config::class, uses = [UserMapper::class])
interface BanMapper : BaseMapper<Ban, BanDto>
